.dockerignore

```diff
@@ -25,3 +25,8 @@ requirements.txt
 
 # Ignore bash scripts
 *.sh
+
+# Ignore secrets
+secrets.txt
```

Comment:
what is secrets.txt? I don't think I see it referenced elswhere in your scripts?

Reply:
fixed.

Reply:
ok, would have liked to know what it was though

Reply:
I copied from .gitignore initially. The initial creator of the .gitignore might have the answer.